spring-cloud-gcp-samples/spring-cloud-gcp-data-jpa-sample/README.adoc

```diff
@@ -4,30 +4,35 @@ This sample application demonstrates how to use
 https://projects.spring.io/spring-data-jpa/[Spring Data JPA] with the
 link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-sql[Spring Cloud GCP SQL Starter].
 
+We will create an instance, a database within the instance, populate the database and then query it.
+
 == Setup
 
-* If you don't have a Google Cloud Platform project with billing enabled, create one from the
-https://console.cloud.google.com/[Google Cloud Console].
+1. Follow https://cloud.google.com/sql/docs/mysql/quickstart[these instructions] to set up a Google
+Cloud Project with billing enabled.
 
-* To authenticate, you have two options:
+2. Create a Google Cloud SQL instance from the
+https://console.cloud.google.com/sql/instances[Google Cloud Console Cloud SQL page].
 
-1. Download the https://cloud.google.com/sdk/[Google Cloud SDK] and run the following command:
-+
-`$ gcloud auth application-default login`
+3. Still within the Google Cloud Console SQL page, create a new database in the instance from the
+"Databases" section. **We recommend you to setup a root password** for your instance (in PostgreSQL you have to).
+**Take note of your root password**, as you will need it in the next step.
 
-2. Create a service account from the
-https://console.cloud.google.com/iam-admin/serviceaccounts[Google Cloud Console], download its
-private key and point the `spring.cloud.gcp.credentials.location` property in the
-`application.properties` file to it.
+4. Open the link:src/main/resources/application.properties[application.properties] file and replace
+*[database-name]* with the name of the database you created in step 3 and
+*[instance-connection-name]* with the instance connection name of the instance you created in
+step 1.
++
+Add the `spring.datasource.password` property with the password as the value.
```

Comment:
Maybe we should put a placeholder for it there, commented out?

Reply:
I'll add the placeholder.
What do you think about adding it for username too?

Reply:
Sure!